### DISCUSSION TITLE
Is there any way to hide the keys I use ?

### DISCUSSION THREAD

**Discussion by Honkeyz** [ppy/osu#19115] [Q&A] [answered]
Is there any way to hide the keys I use ?  (like when start playing or recording liveplay) 
I use for mania QWOP (4k) but i don't want to show that is possible to hide ?

**Comment by peppy** [comment, marked as answer]
Do you mean the key overlay? If so, there's a setting to turn that off during normal gameplay.

**Reply by Honkeyz** [reply]
sorry for taking so long to answer
about the option you said only works for std ? i mean it deactivates the right bar when the game starts
i was trying to ask for something like this
I think I'd better explain with a picture what I wanted to deactivate. 
![screenshot019](https://user-images.githubusercontent.com/108907986/178910287-c59b1ecb-58db-41be-9909-ac3bce00a7b7.jpg)
sorry for wasting your time you can ignore this if you want 
it's a silly question

**Reply by peppy** [reply]
That's not even implemented in lazer. I think you're using stable.

**Reply by Honkeyz** [reply]
yes currently im using stable
but it seems impossible to disable that at the moment anyway thank you very much and sorry for wasting your time.

**Reply by smoogipoo** [reply]
Read https://osu.ppy.sh/wiki/en/Skinning/skin.ini, and see `ColourKeyWarning`. It might support alpha (RGBA) value, contrary to the documentation on that page.

**Reply by Honkeyz** [reply]
I see so its not possible to make invisible / hdie ?

**Reply by smoogipoo** [reply]
As I said in my [comment](https://github.com/ppy/osu/discussions/19115#discussioncomment-3143556), it might work.
